<ui version="4.0" >
 <author>Franz Schmid</author>
 <class>PatternDialog</class>
 <widget class="QDialog" name="PatternDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Patterns</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item>
    <widget class="QListWidget" name="patternView" >
     <property name="flow" >
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonLoad" >
       <property name="text" >
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonLoadDir" >
       <property name="text" >
        <string>Load Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemove" >
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemoveAll" >
       <property name="text" >
        <string>Remove All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>21</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="10" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
